types/queries/insert_chapter_mark_type_genre.sql

```diff
@@ -0,0 +1,17 @@
+insert into collection_chapter (
+  title,
+  document_id,
+  wordpress_id,
+  index_in_parent,
+  chapter_path,
+  section)
+values (
+  $1,
+  ( select id
+    from document
+    where short_name = $2),
+  $3,
+  $4,
+  $5,
+  'Body'
```

Comment:
I couldn't get passing in a CollectionSection as a sixth parameter to work here: I would get the error "unsupported type collection_section for param #6".

Reply:
Hmm it's should be possible for this to work, did you check out usage of `SegmentType`? If you look at line 758 in database_sql.rs, for that custom enum type we have to tell sqlx to ignore type-checking (which is weird, but seems to work) by adding `as _` to the callsite.

Reply:
adding as _ worked, thank you!